Userland/Libraries/LibIDL/Types.h

```diff
@@ -144,13 +144,25 @@ class Type : public RefCounted<Type> {
     bool m_nullable { false };
 };
 
+struct DummyNoncopyable {
+    AK_MAKE_NONCOPYABLE(DummyNoncopyable);
+
+public:
+    DummyNoncopyable() = default;
+    DummyNoncopyable(DummyNoncopyable&&) = default;
+    DummyNoncopyable& operator=(DummyNoncopyable&&) = default;
+};
+
 struct Parameter {
     NonnullRefPtr<Type const> type;
     DeprecatedString name;
     bool optional { false };
     Optional<DeprecatedString> optional_default_value;
     HashMap<DeprecatedString, DeprecatedString> extended_attributes;
     bool variadic { false };
+    [[no_unique_address]] DummyNoncopyable m_dummy {};
```

Comment:
I'd leave `DummyNoncopyable` out entirely - you've tested that it works, let's not confuse others with this.